frontend/src/features/admin-form/create/common/CreatePageSidebarContext/CreatePageSidebarContext.tsx

```diff
@@ -151,6 +154,8 @@ export const useCreatePageSidebarContext =
       setPendingTab(undefined)
     }, [isMobile, pendingTab, setFieldsToInactive])
 
+    const useMeasureDrawer = useMeasure()
```

Comment:
nit: we should not prefix non-hooks with `use` keyword, causes unnecessary confusion. should we call this `drawerDimensions` instead?

Reply:
seems like we only need the drawer width too; suggest just returning `drawerWidth` from this hook instead of the entire dimensions.

Reply:
agree on both points